market/portal/meta.json

```diff
@@ -1,6 +1,13 @@
 {
 	"id": "portal",
 	"name": "Portal",
+	"description": "Powerful workflow user interface",
+	"type": "util",
+	"vendor": "Axon Ivy AG",
+	"costs": "free",
+	"sourceUrl": "https://bitbucket.org/axonivy-prod/axonivyportal/src/master/",
+	"language": "EN/DE/FR/ESP",
```

Comment:
I think the short version for Spanish is just simply "ES"

Reply:
> Good thanks.
> Don't forget to replace the placeholder short descriptions

will be done by balsi